drivers/hmis/app/models/hmis/hud/service.rb

```diff
@@ -28,7 +28,7 @@ class Hmis::Hud::Service < Hmis::Hud::Base
 
   scope :bed_nights, -> { where(RecordType: 200) }
 
-  after_commit :warehouse_trigger_processing
+  after_save :warehouse_trigger_processing
```

Comment:
This one was actually passing tests with `after_commit`, unlike CLS, I'm not sure why. Still updated it though for consistency.